tests/contrib/hooks/test_gcp_api_base_hook.py

```diff
@@ -46,21 +52,70 @@
     default_creds_available = False
 
 
+class TestCatchHttpException(unittest.TestCase):
+    def test_no_exception(self):
+        self.called = False
+
+        class FixtureClass(LoggingMixin):
+            @hook.GoogleCloudBaseHook.catch_http_exception
+            def test_fixutre(*args, **kwargs):
```

Comment:
```suggestion
            def test_fixture(*args, **kwargs):
```
Applicable to all test methods below as well.

Reply:
I apply your suggestion.